### PR TITLE
Ratvarian Spears are more powerful in melee and less easily turned against Servants

### PR DESCRIPTION
**Pull request by ChangelingRain**
:cl: Joan
tweak: Ratvarian Spears now do 18 damage and ignore a small amount of armor on normal attacks, but have a slightly lower chance to knowdown/knockout. Impaling also ignores a larger amount of armor.
rscadd: Throwing a Ratvarian Spear at a Servant will not damage them and may cause them to catch the spear.
/:cl:

### CODE REVIEW

**Comment by Durkel** [comment]
Why do they need stronger spears? Aren't the magical on demand snowflake embed spears already enough? I haven't seen anyone complain about the spears lacking damage so I'm confused why the buff.

**Comment by ChangelingRain** [comment]
Tweaked; damage is now 15 plus 3 burn(which ignores armor) to non-servant targets.

**Comment by PKPenguin321** [comment]
Why stop the crew from turning them against the cult?

**Comment by ChangelingRain** [comment]
They're not entirely stopped; the spear is still an okayish 15 force weapon, as compared to cult blades, which the crew can't use at all.

**Comment by TechnoAlchemisto** [comment]
I don't really think it's necessary to make it even harder to fight back against the cultists when they lose their own weapon. It's not justified with the scarcity and necessity that we see with firing pins in nuke ops weapons either.

**Comment by JJRcop** [comment]
Maybe Ratvar should be disappointed in whatever servant gets hit with the spear that they managed to lose their weapon and get it hurled back at them.

**Comment by ExcessiveUseOfCobblestone** [comment]
>Maybe Ratvar should be disappointed in whatever servant gets hit with the spear that they managed to lose their weapon and get it hurled back at them.
I am a lover of sassy messages ingame!

**Comment by KingPhilipIII** [comment]
You can't deny that many people's downfalls is bullshit one hit disarm/shove before being murdered with your own weapon. I see nothing wrong with this.

**Comment by KorPhaeron** [comment]
@KingPhilipIII 

That's the entire point of disarming though. Everyone is dangerous at any time.

**Comment by AdamElTablawy** [comment]
okay i'm not sure why function call needs a buff seeing as it's basically 'DIE ASSHOLE'-the spell, on command

**Comment by KingPhilipIII** [comment]
Adam, I think you're referring to Ei Nath. 

And it's hardly an ultimate godweapon. It's robust as fuck in the right hands but it has a lengthy cooldown and you're just as liable to get cucked by a taser as you would with any melee weapon. The stun caused by throwing it actually isn't all that long. It's long enough to get up close and hit them a few times, or steal their weapon if you're quick.

**Comment by AdamElTablawy** [comment]
I mean it's certainly long enough to pair judicial visor guvax with it. 

Or just toolbox them to death

**Comment by KingPhilipIII** [comment]
It actually isn't long enough to use the judicial visor, unless the delay got reduced. 

Toolbox them to death, probably yea, since the spear does incredible amounts of damage when thrown,

**Comment by ChangelingRain** [comment]
It CAN be long enough but the stun is 2-4 seconds so about half the time it won't work.

File: code/game/gamemodes/clock_cult/clock_items/ratvarian_spear.dm [hunk 1]
```diff
@@ -45,31 +47,32 @@
 		else
 			impaling = TRUE
 			attack_verb = list("impaled")
-			force += 23 //40 damage if ratvar isn't alive, 53 if he is
+			force += 22 //40 damage if ratvar isn't alive, 50 if he is
```
Comment (RemieRichards) [inline code/game/gamemodes/clock_cult/clock_items/ratvarian_spear.dm]: 37

Comment (ChangelingRain) [inline code/game/gamemodes/clock_cult/clock_items/ratvarian_spear.dm]: I'm counting the bonus 3 burn from attacking someone

**Pull request merged**